common/src/test/resources/application.common.properties

```diff
@@ -12,5 +12,6 @@ spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 
 spring.jpa.hibernate.ddl-auto=none
 spring.liquibase.enabled=true
+spring.liquibase.contexts=none
```

Comment:
Shouldn't this be "test" instead of "none"?

Reply:
See "What Reviewers should Watch For" in the PR comments.

Reply:
I'm still confused. This properties files is in src/test, not src/main so shouldn't it be test according to the explanation?

Reply:
This will be another ticket to switch to test.  I am focused only on the API module for this PR.